app/views/events/participants.html.erb

```diff
@@ -1,40 +1,66 @@
 <%- model_class = Event -%>
+<script type="text/javascript">
```

Comment:
Moreover i would recommend to target specific checkboxes with a specific class to avoid side effects to later implementations.